catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -266,6 +273,15 @@ private SchematronReport generateReport(String metadata, Templates validator)
         return report;
     }
 
+    /**
+     * Replace tabs, literal carriage returns, and newlines with a single whitespace
+     * @param input
+     * @return
+     */
+    static String sanitize(final String input) {
```

Comment:
I could just make it private and indirectly test the sanitization by calling `validate` (which is sort of what's happening in the `testWarningsAndErrorsAreSanitized`. If the regex in `SchematronValidationService` is changed, then it should (in theory) be tested in the `testSanitizationChangesInput` method.